internals/set-parcel-build-variables.js

```diff
@@ -0,0 +1,11 @@
+const buildEnv = Object.create(process.env)
+
+const { buildTime, sha } = require('./getBuildData.js')
+const { csp } = require('./getCsp.js')
+
+buildEnv.REACT_APP_BYPASS_LOCAL = 1
+buildEnv.REACT_APP_BUILD_TIME = buildTime
```

Comment:
This name is confusing me :D  I'd prefer either:
- REACT_APP_BUILD_DATE, buildDate
- REACT_APP_BUILD_DATETIME, buildDatetime